docs/LTS.md

```diff
@@ -24,17 +24,19 @@ Fastify 长期支持计划 (LTS) 以本文档为准：
 | :------ | :----------- | :-------------- | :------------------- |
 | 1.0.0   | 2018-03-06   | 2019-09-01      | 6, 8, 9, 10, 11      |
 | 2.0.0   | 2019-02-25   | 2021-01-31      | 6, 8, 10, 12, 14     |
-| 3.0.0   | 2020-07-07   | 待定            | 10, 12, 14           |
+| 3.0.0   | 2020-07-07   | 待定            | 10, 12, 14, 16       |
 
 <a name="supported-os"></a>
 
 ### 经过 CI (持续集成) 测试的操作系统
 
-| 系统    | 版本                   | 包管理器                  | Node.js   |
-|---------|------------------------|---------------------------|-----------|
-| Linux   | Ubuntu 16.04           | npm                       | 10,12,14  |
-| Linux   | Ubuntu 16.04           | yarn,pnpm                 | 10,12     |
-| Windows | Windows Server 2016 R2 | npm                       | 10,12,14  |
-| MacOS   | macOS X Mojave 10.14   | npm                       | 10,12,14  |
+Fastify 使用 GitHub Actions 来进行 CI 测试，请参阅 [GitHub 相关文档](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources)来获取下表 YAML 工作流标签所对应的最新虚拟机环境。
```

Comment:
nit: use https://docs.github.com/cn/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources